lambda_src/lambda_function.py

```diff
@@ -7,9 +7,12 @@
 from json import dumps, loads
 from typing import Any, Dict, Text
 from urllib.parse import urlparse
+from time import sleep
 
 from .log import format_trace
 from .utils import LOG, create_response, format, invoke_process_lambda
+from .response_storage import (initialize_dynamodb_item, write_dynamodb_item,
```

Comment:
let's not parametrize it just yet, as we have no way of justifying the ned for a user-facing interface just yet.

Reply:
Do you think it might be safer to have it disabled by default, and then enabling it with an optional header? (mainly because of the size limit)

```
....
(other headers)
request-locking = 'dynamodb'
(other headers)
...
```